packages/core/angular/package.json

```diff
@@ -8,25 +8,14 @@
     "publishConfig": {
         "access": "public"
     },
-    "type": "module",
-    "sideEffects": false,
-    "main": "lib/bundles/solana-wallet-adapter-angular.umd.js",
-    "module": "lib/fesm2015/solana-wallet-adapter-angular.js",
-    "types": "lib/solana-wallet-adapter-angular.d.ts",
-    "es2015": "lib/fesm2015/solana-wallet-adapter-angular.js",
-    "esm2015": "lib/esm2015/solana-wallet-adapter-angular.js",
-    "fesm2015": "lib/fesm2015/solana-wallet-adapter-angular.js",
-    "files": [
-        "lib",
-        "src"
-    ],
```

Comment:
Isn't removing all of this a problem for the published package consumers?

Reply:
Hmm, okay. Could this still present issues for other packages in the repo that use this locally? Thinking especially removing the `type": "module"`, sideEffects/main/module/types keys. I guess I'll just try to build it locally and see.

Reply:
`sideEffects`, `module` and `typings` are automatically added by ng-packagr. I believe angular packages don't use a `main` but rather have all those `es2020`, `esm2020`, etc... I do find it curious that the `type: "module"` is not there, it seems like its not needed... Last time I checked it was building properly and I published it under my npm org to check, the repo in the description is using it